README.md

```diff
@@ -87,6 +87,22 @@ Show message with ID 1
 
     RETR 1
 
+## Use with docker-machine
+
+Because service clients, like the addon-administration, that are using the
+3-legged flow to authenticate a user with a redirect to OSIAM only know one
+endpoint to redirect to and back to themselves, this container will be working
+only when forwarding the port of the tomcat who is running inside the docker
+container. Actually this is usable also for any other solution when the
```

Comment:
This solution is actually usable for any other set-up where the container is not running on the host system directly.